client/my-sites/patterns/components/view-toggle/index.tsx

```diff
@@ -5,18 +5,39 @@ import {
 	PatternLibraryToggleOption,
 } from 'calypso/my-sites/patterns/components/toggle';
 import { usePatternsContext } from 'calypso/my-sites/patterns/context';
+import { QUERY_PARAM_SEARCH } from 'calypso/my-sites/patterns/lib/filter-patterns-by-term';
 import { getCategoryUrlPath } from 'calypso/my-sites/patterns/paths';
-import { PatternView } from 'calypso/my-sites/patterns/types';
+import { PatternTypeFilter, PatternView } from 'calypso/my-sites/patterns/types';
 
 import './style.scss';
 
+function getViewToggleUrlPath(
```

Comment:
Did you intentionally not move it to /patterns/paths ?

Reply:
I figured it only applies to this component, so it didn't need to be shared